user-docs/transaction-functions.md

```diff
@@ -0,0 +1,121 @@
+# Transaction Functions available in `algob`
+
+## `getSuggestedParams`
+
+This function performs the task of fetching the common transaction params from the connected network blockchain. Private chains might throw some error while executing this function, please make sure that the node progresses in that case. 
+
+This function accepts:
++ `algoc1`: an instance(object) of `Algodv2` class of `algosdk`, which is to be used to fetch the transaction params from the connected network. 
```

Comment:
```suggestion
+ `algocl`: an instance(object) of `Algodv2` class of `algosdk`, which is to be used to fetch the transaction params from the connected network. 
```

Reply:
oh. will change this.

Reply:
Let's simplify this:
`algocl`: an Algorand client, instance of `Algodv2`, used to communicate with a blockchain node.